src/molgenis-js-client.ts

```diff
@@ -0,0 +1,141 @@
+import fetch from 'cross-fetch'
```

Comment:
why cross fetch ? 
looking at the npm page is says firefox and safari tests are failing